supervisor/resolution/check.py

```diff
@@ -19,9 +18,8 @@ class ResolutionCheck(CoreSysAttributes):
     def __init__(self, coresys: CoreSys) -> None:
         """Initialize the checks class."""
         self.coresys = coresys
-        self._core_security = CheckCoreSecurity(coresys)
-        self._free_space = CheckFreeSpace(coresys)
-        self._addon_pwned = CheckAddonPwned(coresys)
+        self._checks: Dict[str, CheckBase] = {}
+        self.load()
```

Comment:
```suggestion
        self._load()
```

Is not something that should call out of this class or multiple times